internal/Change-DbaSqlServiceStartupMode.ps1

```diff
@@ -1,26 +1,43 @@
 Function Change-DbaSqlServiceStartupMode {
```

Comment:
This is not an approved verb for a command and should be changed to something else....likely `Invoke-...`